dotCMS/src/integration-test/java/com/dotmarketing/portlets/folders/business/FolderAPITest.java

```diff
@@ -844,15 +832,31 @@ public void testFindSubFolders() throws DotDataException, DotSecurityException {
 
 	@Test
 	public void testFindSubFoldersByHost() throws DotDataException, DotSecurityException {
-		final List<Folder> folders = folderAPI.findSubFolders(host, false);
+		final Host newHost = new SiteDataGen().nextPersisted();
+		final String folderPath = "/folder"+System.currentTimeMillis();
```

Comment:
Codacy found an issue: [The String literal "/folder" appears 6 times in this file; the first occurrence is on line 834](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5916479)